### PR TITLE
https://issues.redhat.com/browse/ACM-15363--infra nodes file for 2.12…

### PR DESCRIPTION
**Pull request by swopebe**
… ACMonly

### CODE REVIEW

File: install/config_infra_nodes.adoc [hunk 1]
```diff
@@ -0,0 +1,50 @@
+[#config-infra-node-acm]
+= Configuring infrastructure nodes for {acm-short}
+
+Configure your {ocp-short} cluster to contain infrastructure nodes to run approved {acm-short} management components. Running components on infrastructure nodes avoids allocating {ocp-short} subscription quota for the nodes that are running those management components.
+
```
Comment (swopebe) [inline install/config_infra_nodes.adoc:5]: same thing, but for ACM, only above this line changed.

File: install/install_connected.adoc [hunk 2]
```diff
@@ -260,56 +260,7 @@ oc get mch -o yaml
...
-. Add the following additional configuration before you apply the `MultiClusterHub` custom resource:
-
-+
-[source,yaml]
-----
-spec:
-  nodeSelector:
-    node-role.kubernetes.io/infra: ""
-----
+You can now configure your {ocp-short} cluster to contain infrastructure nodes to run approved management components. Running components on infrastructure nodes avoids allocating {ocp-short} subscription quota for the nodes that are running those management components. See xref:./config_infra_nodes.adoc#config-infra-node-mce[Configuring infrastructure nodes for {acm-short}] for that procedure.
```
Comment (swopebe) [inline install/install_connected.adoc:263]: transition for install completion that may lead to the next step (not required but I will get a sense from Cameron how likely it is that we need to take the user here in our rewrite)

File: install/install_overview.adoc [hunk 3]
```diff
@@ -20,6 +20,7 @@ The documentation references the earliest supported {ocp-short} version, unless
 Installing {acm} sets up a multi-node cluster production environment. You can install {acm} in either standard or high-availability configurations. View the following documentation for more information about the installation and upgrade procedures, as well as information about advanced configuration, scalability, and sizing:
 
 * xref:../install/install_connected.adoc#installing-while-connected-online[Installing while connected online]
+* xref:../config_infra_nodes.adoc#config-infra-node-mce[Configuring infrastructure nodes for {acm-short}]
 * xref:../install/install_disconnected.adoc#install-on-disconnected-networks[Install on disconnected networks]
```
Comment (swopebe) [inline install/install_overview.adoc:24]: Intro

File: install/main.adoc [hunk 4]
```diff
@@ -6,6 +6,7 @@ include::install_overview.adoc[leveloffset=+1]
 include::perform_scale.adoc[leveloffset=+2]
 include::cluster_size.adoc[leveloffset=+3]
 include::install_connected.adoc[leveloffset=+2]
+include::config_infra_nodes.adoc[leveloffset=+2]
```
Comment (swopebe) [inline install/main.adoc:9]: main file add

File: install/config_infra_nodes.adoc [hunk 5]
```diff
@@ -0,0 +1,50 @@
+[#config-infra-node-acm]
+= Configuring infrastructure nodes for {acm-short}
+
+Configure your {ocp-short} cluster to contain infrastructure nodes to run approved {acm-short} management components. Running components on infrastructure nodes avoids allocating {ocp-short} subscription quota for the nodes that are running those management components.
```
Comment (dockerymick) [inline install/config_infra_nodes.adoc]:
```suggestion
Configure your {ocp-short} cluster to contain infrastructure nodes to run approved {acm-short} management components. Running components on infrastructure nodes avoids allocating {ocp-short} subscription quota for the nodes that are running {acm-short} management components.
```

File: install/config_infra_nodes.adoc [hunk 6]
```diff
@@ -0,0 +1,50 @@
...
+[source,yaml]
+----
+metadata:
+  labels:
+    node-role.kubernetes.io/infra: ""
+spec:
+  taints:
+  - effect: NoSchedule
+    key: node-role.kubernetes.io/infra
+----
```
Comment (dockerymick) [inline install/config_infra_nodes.adoc]:
```suggestion
Follow the procedures that are described in link:https://docs.redhat.com/documentation/en-us/openshift_container_platform/4.15/html/machine_management/creating-infrastructure-machinesets[Creating infrastructure machine sets] in the {ocp-short} documentation. Infrastructure nodes are configured with a Kubernetes `taints` and `labels` to keep non-management workloads from running on them.

. To be compatible with the infrastructure node enablement provided by {acm-short}, ensure your infrastructure nodes have the following `taints` and `labels` applied:

+
[source,yaml]
----
metadata:
  labels:
    node-role.kubernetes.io/infra: ""
spec:
  taints:
  - effect: NoSchedule
    key: node-role.kubernetes.io/infra
----
```

File: install/install_overview.adoc [hunk 7]
```diff
@@ -20,6 +20,7 @@ The documentation references the earliest supported {ocp-short} version, unless
 Installing {acm} sets up a multi-node cluster production environment. You can install {acm} in either standard or high-availability configurations. View the following documentation for more information about the installation and upgrade procedures, as well as information about advanced configuration, scalability, and sizing:
 
 * xref:../install/install_connected.adoc#installing-while-connected-online[Installing while connected online]
+* xref:../config_infra_nodes.adoc#config-infra-node-mce[Configuring infrastructure nodes for {acm-short}]
```
Comment (dockerymick) [inline install/install_overview.adoc]: This link appeared as broken 
Screenshot 2024-12-02 at 2 24 47 PM

```suggestion
* xref:../install/config_infra_nodes.adoc#config-infra-node-mce[Configuring infrastructure nodes for {acm-short}]
```

**Review by dockerymick — CHANGES_REQUESTED** [review]

File: install/config_infra_nodes.adoc (hunk 5 above)
Comment (swopebe) [reply install/config_infra_nodes.adoc]: I had committed right before, so this is in. TY!

File: install/config_infra_nodes.adoc (hunk 6 above)
Comment (swopebe) [reply install/config_infra_nodes.adoc]: Got this right before, was trying to save you time. TY!

File: install/install_connected.adoc (hunk 2 above)
Comment (dockerymick) [reply install/install_connected.adoc:263]: On line 20, there is a link to "Cleaning up artifacts before reinstalling". I know it it wasn't changed for this PR, but will you still look into updating that link reference? 

I can also create a new issue if you prefer that

Comment (swopebe) [reply install/install_connected.adoc:263]: I planned to do it here. I mentioned on Slack, but I cannot find out why it is not working. 

[2:12](https://redhat-internal.slack.com/archives/GVDCLR7A5/p1733166730802879)
(there is a broken link on one file in the book -- not related to this. I will address that before merging) ^^

**Review by dockerymick — APPROVED** [review]

**Pull request merged**